en/docs/Administer/ProductSecurity/enabling-java-security-manager.md

```diff
@@ -1,157 +1,159 @@
-                Signature algorithm name: SHA1withRSA
-                Version: 3
-                Trust this certificate? [no]:  yes
-                Certificate was added to keystore
+    Then, import the same `signFiles` certificate to the `wso2carbon.jks` keystore by executing the command given below. Be sure to specify the correct directory path to the `wso2carbon.jks` file of your product.
+
+    ```bash
+    $ keytool -import -alias signFiles -file sign-cert.cer -keystore <PATH_to_PRODUCT_HOME>/repository/resources/security/wso2carbon.jks
+    Enter keystore password:  
+    Owner: CN=John, OU=Engineering, O=WSO2, L=Colombo, ST=Western, C=LK
+    Issuer: CN=John, OU=Engineering, O=WSO2, L=Colombo, ST=Western, C=LK
+    Serial number: 5486f3b0
+    Valid from: Tue Dec 09 18:35:52 IST 2014 until: Fri Dec 06 18:35:52 IST 2024
+    Certificate fingerprints:
+    MD5:  54:13:FD:06:6F:C9:A6:BC:EE:DF:73:A9:88:CC:02:EC
+    SHA1: AE:37:2A:9E:66:86:12:68:28:88:12:A0:85:50:B1:D1:21:BD:49:52
+    Signature algorithm name: SHA1withRSA
+    Version: 3
+    Trust this certificate? [no]:  yes
+    Certificate was added to keystore
     ```
 
-        !!! note
-    Note that WSO2 no longer recommends MD5 for JAR signing due to cryptographic limitations.
+    !!! note
+        Note that WSO2 no longer recommends MD5 for JAR signing due to cryptographic limitations.
 
 
-4.  Open the security policy file, and update the "grant signedBy" value in the with the new `signFiles` alias key, as shown below.
+5.  Open the security policy file, and update the "grant signedBy" value in the with the new `signFiles` alias key, as shown below. If security policy file is not created before, create a `sec.policy` file in the `<PRODUCT_HOME>/repository/conf` folder and add the following content to this file.
 
-    ``` java
-        grant signedBy "signFiles" {
-          // permission java.util.PropertyPermission "*", "read";
-          // permission java.lang.RuntimePermission "*", "*";
-          // permission java.io.FilePermission "*", "*";
-          permission java.security.AllPermission;
-        };
+    ``` text
+    grant signedBy "signFiles" {
+      // permission java.util.PropertyPermission "*", "read";
+      // permission java.lang.RuntimePermission "*", "*";
+      // permission java.io.FilePermission "*", "*";
+      permission java.security.AllPermission;
+    };
     ```
 
-5.  Prepare the scripts to sign the JARs and grant them the required permission. For example, the `signJar.sh` script given below can be used to sign each JAR file separately or you can use the `signJars.sh` script, which runs a loop to read all JARs and sign them.
+6.  Prepare the scripts to sign the JARs and **grant them the required permission**(ex: `sudo chmod 755 signJar.sh`). For example, the `signJar.sh` script given below can be used to sign each JAR file separately or you can use the `signJars.sh` script, which runs a loop to read all JARs and sign them.
```

Comment:
The abbreviation that we use in WSO2 docs for example is not ex: it should be e.g.,

Reply:
Fixed in f906106